### PR TITLE
Build a wheel for manylinux

### PR DESCRIPTION
**Pull request by hardbyte**
This PR gets travis to build a wheel file for manylinux as defined in [pep-0513](https://www.python.org/dev/peps/pep-0513/) based off https://github.com/pypa/python-manylinux-demo

The `.travis.yml` file instructs Travis to run the script `travis/build-wheels.sh` inside of a 64-bit manylinux docker build environment. This script builds the anonlink package using pip. 

As these wheels link against an external library the build script runs the wheels through `auditwheel` to create a self-contained wheels. This copies the external library into the wheel itself, so that users won't need to install any extra non-PyPI dependencies (in theory...)

Ideally travis would then upload this wheel to [S3](https://docs.travis-ci.com/user/uploading-artifacts/) or release on github/pypi

### CODE REVIEW

File: travis/build-wheels.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e -x
+
+yum install -y atlas-devel
+
+# Compile wheels
+for PYBIN in /opt/python/cp37-cp37m/bin; do
```
Comment (unzvfu) [inline travis/build-wheels.sh]: Here and below on line 19 the code iterates over a list with one element, which serves no purpose. I presume that you're anticipating a time in which there will be several Python binary prefix paths and that you'll need to iterate over them; if that's the case, you should have a separate variable like `PYBIN_PATHS=/opt/python/cp37-cp37m/bin` and iterate over it on lines 7 and 19 with `for PYBIN in $PYBIN_PATHS;`. (You could set `PYBIN_PATHS` with `?=` if you want to optionally set it in the calling environment. You can use `tr ':' ' '` if you want `PYBIN_PATHS` to look like a normal colon-separated list of paths.)

Comment (unzvfu) [reply travis/build-wheels.sh]: Additionally, I guess the `PYBIN` path come from the manylinux Docker image. It looks suspiciously specific. Is there really no more general path? Or is it encoding a Python version perhaps? If the latter, perhaps it's worth mentioning that?

File: .travis.yml [hunk 2]
```diff
@@ -18,6 +15,19 @@ matrix:
     - python: 'nightly'
     - python: 'pypy3'
 
+jobs:
+  include:
+    - stage: Build wheels
+      sudo: required
+      python: '3.6'
+      services:
+      - docker
```
Comment (unzvfu) [inline .travis.yml]: Nitpick: Missing two indent spaces.

File: travis/build-wheels.sh (hunk 1 above)
Comment (hardbyte) [reply travis/build-wheels.sh]: All good points. Will remove the looping and add a link to the manylinux project which talks about the paths

**Pull request merged**